build.gradle

```diff
@@ -55,16 +55,16 @@ dependencies {
 	implementation 'com.github.jonpeterson:jackson-module-model-versioning:1.2.2'
 	implementation 'org.apache.commons:commons-math3:3.6.1'
 	implementation 'org.apache.commons:commons-lang3:3.12.0'
-	implementation 'net.java.dev.jna:jna:5.10.0'
-	implementation 'net.java.dev.jna:jna-platform:5.10.0'
+	implementation 'net.java.dev.jna:jna:5.12.1'
```

Comment:
If we want to keep the deps up to date, maybe it would be better if we use are version range prefix? Targeting latest minor version should be fine like that `5.+`. Same for everything else.